finetune/target_models/classifier.py

```diff
@@ -68,20 +60,22 @@ def featurize(self, X, **kwargs):
         """
         return super().featurize(X, **kwargs)
 
-    def predict(self, X, probas=False, context=None, **kwargs):
+    def predict(self, X, context=None, **kwargs):
         """
         Produces a list of most likely class labels as determined by the fine-tuned model.
         :param X: list or array of text to embed.
         :returns: list of class labels.
 
-
         Chunk idx for prediction.  Dividers at `step_size` increments.
         [  1  |  1  |  2  |  3  |  3  ]
         """
+        return super().predict(X, context=context, **kwargs)
+
+    def _predict_internal(self, zipped_data, probas=False, **kwargs):
```

Comment:
I don't understand the distinction between `_predict` and `_predict_internal`.  From what I see so far `_predict` just calls `_predict_internal`?